cms/static/js/views/utils/create_library_utils.js

```diff
@@ -1,58 +1,22 @@
 /**
  * Provides utilities for validating libraries during creation.
  */
-define(["jquery", "underscore", "gettext", "js/views/utils/view_utils"],
-    function ($, _, gettext, ViewUtils) {
+define(["jquery", "underscore", "gettext", "js/views/utils/view_utils", "js/views/utils/create_utils_base"],
+    function ($, _, gettext, ViewUtils, CreateUtilsFactory) {
         "use strict";
-        return function (selectors, classes) {
-            var validateTotalKeyLength, setNewLibraryFieldInErr, hasInvalidRequiredFields,
-                createLibrary, validateFilledFields, configureHandlers;
+        var CreateLibrary = function (selectors, classes) {
+            CreateUtilsFactory.apply(this);
```

Comment:
Sorry to be a pain, but still trying to understand this "apply" usage. What does it do that CreateLibrary.prototype = CreateUtilsFactory (below) does not do?